### DISCUSSION TITLE
Litebug: rapid data entry interface for firefly iii

### DISCUSSION THREAD

**Discussion by andho** [firefly-iii/firefly-iii#5923] [General]
Hi all,

I just published https://github.com/andho/litebug. It's a very simple form that allows for rapid data entry. The README will include more details. It uses PAT for auth. When firefly iii get's oauth PKCE flow, I'm going to implement it. Currently the biggest issue for most might be setting up CORS on their firefly instances, but other than that, it should be easy to use this with your firefly instance.

**Comment by JC5** [comment]
Cool! Would you like it to be featured in the documentation somewhere? Twitter?

**Reply by andho** [reply]
Not yet. Maybe after some of more of the features. Thanks for the offer.

**Reply by JC5** [reply]
Let me know, good stuff!
